CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.2.0 (2020-07-08)
+
+- Added the ability to disable the autocomplete component via a reactive form. [#101](https://github.com/blackbaud/skyux-lookup/pull/101)
```

Comment:
Should we reword this to point out the new `@Input()` property? Something like ...
```
Added the `disabled` input property to the the `skyAutocomplete` directive.
```

Reply:
I like that, but I don't think we need `input`.

Reply:
@Blackbaud-TrevorBurch, I'm going to go ahead and make this change. I can always change it back if you it's a problem.